cdap-ui/app/cdap/components/CaskWizards/ArtifactUpload/UploadStep/index.js

```diff
@@ -46,21 +43,12 @@ const ArtifactUploader = connect(
   mapDispatchWithDNDFileProps
 )(FileDnD);
 
-export default function UploadStep(undefined, context) {
+export default function UploadStep() {
   return (
     <Provider store={ArtifactUploadStore}>
       <div className="upload-step-container">
-        {/* TODO: shouldn't do this, replace in 4.2} */
-        context.isMarket ? (
-          <h4 className="upload-instruction">
-            {T.translate('features.Wizard.ArtifactUpload.Step1.uploadHelperText')}
```

Comment:
we're removing this?

Reply:
Yeah it is never getting used today. In hub we render the same message as we do for plus button modals.